packages/compiler/test/tsx/non-latin.ts

```diff
@@ -66,3 +66,5 @@ export default function __AstroComponent_(_props: Record<string, any>): any {}`;
   const { code } = await convertToTSX(input, { sourcemap: 'external' });
   assert.snapshot(code, output, `expected code to match snapshot`);
 });
+
+test.run();
```

Comment:
is this a requirement of uvu tests ig?

Reply:
Yeah, I'm not sure if these were still running, but better safe than sorry!